pages/chaoss/visualizations/project_velocity.py

```diff
@@ -0,0 +1,352 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.contributors_query import contributors_query as ctq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+import math
+import numpy as np
+
+"""
+(5) CONTEXT OF GRAPH
+
+"""
+
+
+PAGE = "chaoss"
+VIZ_ID = "project-velocity"
+
+gc_project_velocity = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Project Velocity",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody("INSERT CONTEXT OF GRAPH HERE"),
```

Comment:
Missed this

Reply:
@JamesKunstle  Do you have any wording suggestions for this? Thinking about it a bit but would be great to have input as well since this graph is so complex

Reply:
Thankfully, the graph itself is pretty descriptive. I think you could link the CHAOSS definition, and that'd be pretty solid. My expectation is that most people will get the vibe pretty quickly. 

I'd maybe also add a recommendation to strongly consider filtering on the last 6-12 months. Velocity is instantaneous, as should be the visualization- e.g. Kubernetes would have a crazy velocity if we take its entire history into account 😃

Reply:
Hm I see what you are saying about the time range. One thing that I am worried about with that is that people will assume the base visualization is all time without actually checking vs them having to select the time range themselves for a window view. Most of the applications of project velocity I see are all time